configure.ac

```diff
@@ -2255,7 +2255,7 @@ AC_CONFIG_COMMANDS_PRE([
   mkexe_exp="$mkexe_cmd_exp"
   AS_IF([test -n "$mkexe_cflags"],
     [mkexe="$mkexe $mkexe_cflags"
-    mkexe_exp="$mkexe_exp $oc_cflags $CFLAGS"])
+    mkexe_exp="$mkexe_exp $common_cflags $CFLAGS"])
```

Comment:
For the sake of example, a typical `$common_cflags` is `-O2 -fno-strict-aliasing -fwrapv` and a typical `$oc_cflags` adds some warnings, some code generation options like `-fno-common`, and `-g`.